lib/service/marker_helper.dart

```diff
@@ -2,11 +2,12 @@ import 'package:concordi_around/model/coordinate.dart';
 import 'package:flutter/material.dart';
 import 'package:google_maps_flutter/google_maps_flutter.dart';
 import 'package:concordi_around/data/data_points.dart' as data;
+import 'package:concordi_around/widget/map.dart';
```

Comment:
At first I was worried that a map.dart dependency was added; however, this import is unused. Can you please remove, if unnecessary?

Reply:
I removed the unused import